### PR TITLE
Add margin to top of story promo list

### LINKED ISSUE #662: No spacing between banner and StoryPromo list when screen width <600px
**Describe the bug**
On updating Simorgh FrontPageSection container to use the latest version of @bbc/psammead-section-label, there is now no space between the banner and the first story of the first section

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'http://localhost:9001/?path=/story/front-page--igbo'
2. Reduce screen width to <600px

**Expected behaviour**
There should be adequate space between the bottom of the Banner and the top of the first story.

8px when <400px
16px when <600px
0 when >600px (as section header shows)

**Solution Option 1:**
Put the margin on the Unordered list

When you put the margin on the UL, when under a visible section label the margins just overlap, as if the UL margin isnt there. When there isnt a visible section label though, it uses the margin on the top as expected.

**Bug Screenshots:**

Screenshot 2019-06-20 at 11 15 01
**Additional context**

### LINKED ISSUE #670: Confirm brand padding between 400 & 600px
**Is your feature request related to a problem? Please describe.**
The current brand implementation has 16px padding untill it drop below 600px wide, when it goes to 8px. This doesnt match the page wrapper, which drops from 16px to 8px at 400px wide.

This causes a 200px buffer, between 400px and 600px wide where the paddings dont match.

image

The current zeplin does say this styling is correct, as seem in the image below, it just looks weird IMO so thought we should check

**Describe the solution you'd like**
~Confirm with UX that this is expected, if so, close the issue. If not, fix it :P~

Confirmed with UX, the brand should go to 16px padding at 400px instead of 600px

**Testing notes**
X-Browser testing only

Dev insight: Unit snapshots

### PR DESCRIPTION
**Pull request by dr3**
Resolves #662

**Overall change:** Adds margin to Unordered list to fix issue when not under a visible section label

Relies on Margin overlapping to not affect styling when section label is visible

**Code changes:**

- Adds top margin 8px below 400px, 16px above

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-story-promo-list/src/index.jsx [hunk 1]
```diff
@@ -36,8 +37,12 @@ const StyledListItem = styled.li`
 
 const StyledUnorderedList = styled.ul`
   list-style-type: none;
-  margin: 0;
+  margin: ${GEL_SPACING} 0 0 0;
```
Comment (DenisHdz) [inline packages/components/psammead-story-promo-list/src/index.jsx:40]: Is UX happy with this separation if there is no `SectionLabel`. 
At the moment `SectionLabel` uses a margin-top of `1.5rem`, but you won't be able to overlap with that value.

Comment (dr3) [reply packages/components/psammead-story-promo-list/src/index.jsx:40]: Yep, 16px and 8px to match the side padding

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Review by j-pendlebury — APPROVED** [review]

**Comment by david-boydell** [comment]
Looks good, this is ready for merge.

**Pull request merged**